attestation/attestation.go

```diff
@@ -25,15 +25,15 @@ type Report struct {
 // ErrEmptyReport is returned by VerifyRemoteReport if reportBytes is empty.
 var ErrEmptyReport = errors.New("empty report")
 
-// VerifyAzureAttestationToken takes a Microsoft Azure Attestation Token in JSON Web Token compact
-// serialization format and verifies the tokens public claims and signature. The Attestation providers
-// keys are loaded from providerURL/certs over TLS and need to be in JSON Web Key format. The validation
-// is based on the trust in this TLS channel. Note, that the token's issuer (iss) has to equal the providerURL,
-// and providerURL must use the HTTPS scheme.
+// VerifyAzureAttestationToken takes a Microsoft Azure Attestation token in JSON Web Token compact
+// serialization format and verifies the token's public claims and signature. The attestation provider's
+// keys are loaded from providerURL over TLS. The validation is based on the trust in this TLS channel.
```

Comment:
Shouldn't we mention that the path has to be "/certs"?

Reply:
Microsoft is the only party that can operate an attestation server, so I thought that this is an implementation detail the user should not worry about.

Reply:
Okay, I'm not so familiar with Azure, I thought you could choose your own path if you create your own provider as mentioned in this line:
[>  or create their own providers with custom policies.](https://docs.microsoft.com/en-us/azure/attestation/basic-concepts#regional-shared-provider)